mava/advanced_usage/ff_ippo_store_experience.py

```diff
@@ -31,8 +31,8 @@
 from rich.pretty import pprint
 
 from mava.evaluator import get_eval_fn, make_ff_eval_act_fn
-from mava.networks import FeedForwardActor as Actor
-from mava.networks import FeedForwardValueNet as Critic
+from mava.networks.base import FeedForwardActor as Actor
+from mava.networks.base import FeedForwardValueNet as Critic
```

Comment:
Nit: maybe import these into `networks/__init__.py` then you don't need to go `networks.base`

Reply:
Done :)